lib/cocoapods/sources_manager.rb

```diff
@@ -224,6 +272,11 @@ def update(source_name = nil, show_output = false)
             check_version_information(source.repo)
           end
         end
+        # Perform search index update operation as a subprocess.
+        fork do
```

Comment:
should the forked process do `Process.daemon` so that it doesn't keep the parent process alive?